aim/sdk/adapters/pytorch.py

```diff
@@ -62,12 +62,14 @@ def get_model_layers(model, dt, parent_name=None):
             layers[layer_name] = {}
             if hasattr(m, 'weight') \
                     and m.weight is not None \
-                    and hasattr(m.weight, dt):
+                    and hasattr(m.weight, dt) \
```

Comment:
can lines 65-66 be rewritten as `getattr(m.weight, dt, None) is not None`? 🤔

Reply:
Yup. Committed the requested changes.